apps/site/src/pages/settings/billing-settings-panel/free-or-hobby-subscription-tier-overview.tsx

```diff
@@ -457,7 +484,10 @@ export const FreeOrHobbySubscriptionTierOverview: FunctionComponent<{
             Best for embedders, devs and power-users
           </Typography>
           <LinkButton
-            href={{ pathname: "/upgrade", query: { tier: "pro" } }}
+            href={{
+              pathname: "/settings/billing/upgrade",
+              query: { tier: "pro" },
+            }}
```

Comment:
Not introduced in this PR but this button should probably also read `Continue` instead of `Upgrade` when the user is free.

Reply:
You're right, this was an oversight. Have addressed as a drive-by in https://github.com/blockprotocol/blockprotocol/pull/901/commits/a62b19152b03a8f20084404672a1200edd65eda3